tests/src/e2e/eth_bridge_tests.rs

```diff
@@ -6,45 +8,41 @@ use crate::e2e::setup::constants::{
 use crate::e2e::setup::{Bin, Who};
 use crate::{run, run_as};
 
-const ETH_BRIDGE_ADDRESS: &str = "atest1v9hx7w36g42ysgzzwf5kgem9ypqkgerjv4ehxgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpq8f99ew";
-
 /// # Examples
 ///
 /// ```
 /// let storage_key = storage_key("queue");
 /// assert_eq!(storage_key, "#atest1v9hx7w36g42ysgzzwf5kgem9ypqkgerjv4ehxgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpq8f99ew/queue");
 /// ```
 fn storage_key(path: &str) -> String {
-    format!("#{ETH_BRIDGE_ADDRESS}/{}", path)
+    format!("#{}/{}", eth_bridge::vp::ADDRESS, path)
```

Comment:
is this the same as `namada_core::ledger::eth_bridge::ADDRESS`?

Reply:
I think so but `namada_core::ledger::eth_bridge::ADDRESS` isn't present in this PR. This PR is a rework of a small cleanup PR (https://github.com/anoma/namada/pull/323) which never got merged